sysvabi64/sysvabi64.rst

```diff
@@ -916,10 +923,639 @@ These follow the AAELF64_ base definition from Arm.
 
    PageBreak oneColumn
 
-Program Linking and Dynamic Linking
+Program Loading and Dynamic Linking
 ===================================
 
-Not available for this Alpha release.
+The system works on creating a program/process image by logically
+copying or mapping the program image from storage into a virtual
+memory segment. In most systems with demand paging, when the actual
+page is mapped into physical memory depends on the execution behaviour
+of the program and the working set that it requires for its correct
+execution. To minimize the number of such reads from the physical
+medium containing the program, executables and shared objects must
+have program segments whose file offsets and virtual addresses are
+congruent modulo the page size.
```

Comment:
In ELF the relationship is `p_vaddr` is congruent to `p_offset` (modulo `p_align`). It gets a bit complicated with the relationship between `p_align` and the page size as we have max-page-size which is set to the maximum possible supported page size (64 KiB) and common-page-size which is often set to (4 KiB) I had originally wrote that `p_align` should be set to the max-page-size but it turns out (https://sourceware.org/bugzilla/show_bug.cgi?id=28689) that this is overly strict as some dynamic linkers can adjust based on what platform they are running on to avoid overalignment.

I'll try and find a better way of wording it.

I'll clarify the wording. A loadable segment is used in http://www.sco.com/developers/gabi/latest/ch5.pheader.html to refer to a segment described by a program header with type `PT_LOAD` 
Checking ELF the requirements for non `PT_LOAD` program headers are stricter, they should be `p_vaddr` = `p_offset` (modulo `p_align`) although typically the `p_align` is only 4 or 8 for non loadable segmentes.